source/vibe/web/common.d

```diff
@@ -319,51 +337,29 @@ PathAttribute path(string data)
 
 
 /**
-	UDA to define root URL prefix for annotated REST interface.
-	Empty path means deducing prefix from interface type name (see also rootPathFromName)
+	Deprecated. Use @$(D path) instead.
  */
-RootPathAttribute rootPath(string path)
+deprecated(`Use @path("path") directly.`)
```

Comment:
`deprecated` still there? Do you still have objections against the three-phase deprecation?